src/core/src/core_logic/ExecutionConfig.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, env_layer, composite_logger, execution_parameters):
         self.excluded_package_name_mask_list = self.__get_execution_configuration_value_safely(self.config_settings, Constants.ConfigSettings.PATCHES_TO_EXCLUDE, [])
```

Comment:
Not reading and storing cloudtype here?

Reply:
CloudType is actually in the other PR as discussed. This made it into this PR due to the situation with healthstore id.